### ISSUE TITLE
Retargeting question

### ISSUE THREAD

**Issue by chaserat** [DeepMotionEditing/deep-motion-editing#74]
I use intra-structure retargeting. 
I change the "input.bvh", while the "gt.bvh" remains the original one. But the "result.bvh" style is almost the "gt.bvh" but not "input.bvh". Is it right?  I'm very confused of this result. Any Ideas?

**Comment by PeizhuoLi** [comment]
If you are using the demo we provided, the input.bvh is copied from the specified file for demonstration. Changing input.bvh in the result directory doesn't make any change to the input of the retargeting processing.

Currently we don't have a script to perform an end-to-end retargeting. If you want to test it I'd suggest you to read the demo.py file.

**Comment by chaserat** [comment]
Does it mean the "fix_foot_contact" function in  "models.IK" acts well with "gt.bvh" & "result.bvh"? Actually, the "input.bvh" is still needed in "demp.py".

**Comment by PeizhuoLi** [comment]
fix_foot_contact extracts foot contact information from input.bvh, so yes, it needs input.bvh. As shown in the code, every time you run the demo.py, the input.bvh is copied from the specified file before any further process. My point is that changing input.bvh in the result directory doesn't have any effect to the "real" input of the retargeting.

**Comment by chaserat** [comment]
Yes,.Interestingly, I change the original file in the original directory in "datasets\Mixamo\..", but the result is far from the "input.bvh". 
I was wondering that in "fix_foot_contact",  "input.bvh", "gt.bvh" is from the original directory. And "eval_single_pair.py" process in "demo.py" seems no use for these two files, is it right?

**Comment by PeizhuoLi** [comment]
Not quite exactly. In eval_singal_pair.py, line 84, it reads the specified file. It should work. We are planning to release a script that directly deal with retargeting from one bvh to another soon. Maybe that will help.
